packages/pages/src/dev/server/ssr/generateTestData.ts

```diff
@@ -208,26 +209,27 @@ async function spawnTestDataCommand(
   });
 }
 
-const getSiteStream = (projectStructure: ProjectStructure) => {
+export const getSiteStream = (
+  projectStructure: ProjectStructure
+): SiteStreamConfig | undefined => {
   const siteStreamPath = path.resolve(
     projectStructure.getSitesConfigPath().path,
     projectStructure.config.sitesConfigFiles.siteStream
   );
 
   if (fs.existsSync(siteStreamPath)) {
-    return prepareJsonForCmd(
-      JSON.parse(fs.readFileSync(siteStreamPath).toString())
-    );
+    return JSON.parse(fs.readFileSync(siteStreamPath).toString());
```

Comment:
I think this wouldn't return data in the same format. We have a migration command in migrateConfig.ts call `formatSiteStream` which removes the $ from id. So if someone was on the old sites-config version then the format wouldn't work. I think we need to run it through this.

I'm thinking we move this function to src/common/src/feature/stream.ts and rename it to readSiteStream. We should define another type in there for this version of a SiteStream, which is different from StreamConfig. Add a comment about it. Additionally, if using sites-config, we should use that formatSiteStream on it. Move that function to this file as well. Then update all callers. Basically a little bit of refactoring work.